src/python/pants/backend/python/lint/isort/rules_integration_test.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from typing import List, Optional, Tuple
+
+from pants.backend.python.lint.isort.rules import IsortSetup, fmt, lint, setup_isort
+from pants.backend.python.lint.isort.subsystem import Isort
+from pants.backend.python.rules.download_pex_bin import download_pex_bin
+from pants.backend.python.rules.pex import CreatePex, create_pex
+from pants.backend.python.subsystems.python_native_code import (
+  PythonNativeCode,
+  create_pex_native_build_environment,
+)
+from pants.backend.python.subsystems.python_setup import PythonSetup
+from pants.backend.python.subsystems.subprocess_environment import (
+  SubprocessEnvironment,
+  create_subprocess_encoding_environment,
+)
+from pants.backend.python.targets.formattable_python_target import FormattablePythonTarget
+from pants.build_graph.address import Address
+from pants.engine.fs import Digest, FileContent, InputFilesContent, Snapshot
+from pants.engine.legacy.structs import TargetAdaptor
+from pants.engine.rules import RootRule
+from pants.engine.selectors import Params
+from pants.rules.core.fmt import FmtResult
+from pants.rules.core.lint import LintResult
+from pants.source.wrapped_globs import EagerFilesetWithSpec
+from pants.testutil.subsystem.util import global_subsystem_instance, init_subsystems
+from pants.testutil.test_base import TestBase
+
+
+class IsortIntegrationTest(TestBase):
+
+  good_source = FileContent(path="test/good.py", content=b'from colors import blue, green\n')
```

Comment:
I would make this different from fixed bad to remove the possibility of accidental confusion by using the wrong file. For instance, add `red` at the end for this one only or go for `from animals import cat, dog`

Reply:
Good suggestion!